src/main/java/BasicJavaHomeWork/Exercise_3.java

```diff
@@ -0,0 +1,40 @@
+package BasicJavaHomeWork;
+import java.util.Scanner;
+
+class Exercise_3 {
```

Comment:
Same here, please do this for all classes

Reply:
done